core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -74,6 +74,25 @@
           <div class="protractor-test-conversation-input"
                angular-html-bind="activeCard.interactionHtml">
           </div>
+          <div ng-if="currentInteractionHints.length" style="text-align: right">
+            <span ng-if="areAllHintsExhausted()" tooltip="<['I18N_PLAYER_HINTS_EXHAUSTED' | translate]>">
```

Comment:
Too much duplication with the previous file. Instead, pass allHintsAreExhausted and currentHintIsAvailable into the hint-button directive and branch there.